src/src/hooks/Tracking.js

```diff
@@ -0,0 +1,18 @@
+import { useContext, useState } from "react";
+import TrackingContext from 'TrackingContext';
+
+export function useTracking(opts) {
+  const { trackingIsEnabled } = useContext(TrackingContext);
+
+  let track = (...opts) => {
+    if (trackingIsEnabled) {
+      // eslint-disable-next-line no-undef
+      _paq.push(['trackEvent', ...opts]);
```

Comment:
Where does the _paq come from?
It seems magical that it just is here and is available 😨